### PR TITLE
Add pulp squeezer 0.0.7 announcement

### PR DESCRIPTION
**Pull request by melcorr**
I added you as co-author @mdellweg since you're my co-author :) 

https://melcorr.github.io/pulpproject.org/2021/02/08/pulp-squeezer-0.0.7/

### CODE REVIEW

File: _posts/2021-02-08-pulp-squeezer-0.0.7.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+* `deb_distribution`
+* `deb_publication`
+* `deb_remote`
+* `deb_repository`
+* `deb_sync`
+
+For the `deb_sync` module, you might want to register the sync as a [handler](https://docs.ansible.com/ansible/latest/user_guide/playbooks_handlers.html).
+
+This release also includes the automatic refreshing of the API docs in the status module if version discrepancies are detected.
```
Comment (mdellweg) [inline _posts/2021-02-08-pulp-squeezer-0.0.7.md:25]: :+1:

File: _posts/2021-02-08-pulp-squeezer-0.0.7.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+As part of this release, you can now use Pulp Squeezer to automate workflows with the following new modules:
+
+* `deb_distribution`
+* `deb_publication`
+* `deb_remote`
+* `deb_repository`
+* `deb_sync`
+
+For the `deb_sync` module, you might want to register the sync as a [handler](https://docs.ansible.com/ansible/latest/user_guide/playbooks_handlers.html).
```
Comment (mdellweg) [inline _posts/2021-02-08-pulp-squeezer-0.0.7.md]: Ok, let's pump this up a little. ;) Having an ansible article without the word "idempotent" feels rummy.

```suggestion
The first four in the list, associated with actual resources, provide the `state` parameter to allow for idempotent playbooks the ansible way.
The `deb_sync` module however, being an action module, you might want to register as a [handler](https://docs.ansible.com/ansible/latest/user_guide/playbooks_handlers.html).
```

**Review by mdellweg — COMMENTED** [review]
Now i need to deliver... :nerd_face:

**Comment by melcorr** [comment]
If you're happy with this now, we can merge and add to the Bullhorn!

**Review by mdellweg — APPROVED** [review]
I am happy! Thanks for writing it up.

**Pull request merged**